pkg/debugapi/version.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2021 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package debugapi
+
+// Version is set in the build process.
+var Version string
```

Comment:
here too

Reply:
This doesn't need to but just for the symmetry with the `api` package, I'd also export it.